enginefs.js

```diff
@@ -201,6 +203,23 @@ router.all("/:infoHash/create", function(req, res) {
         res.end(JSON.stringify(getStatistics(engines[ih])));
     });
 });
+
+router.all("/create", function(req, res) {
+    var torrent = fs.readFile(req.body.from,function (error, file) {
+        if (error){
+            res.writeHead(400, jsonHead);
+            res.end();
+            console.error(error)
+            return
+        } 
+        var parsed = parseTorrentFile(file)
```

Comment:
maybe this should be in a try-catch, but i'm not sure; check if the underlying implementation can throw